src/mcp-bundle/config/services.php

```diff
@@ -21,18 +24,27 @@
             ->args(['mcp'])
             ->tag('monolog.logger', ['channel' => 'mcp'])
 
+        ->set('ai.mcp.registry.inner', Registry::class)
+            ->args([service('event_dispatcher'), service('monolog.logger.mcp')])
+
+        ->set('ai.mcp.registry', TraceableRegistry::class)
+            ->args([service('ai.mcp.registry.inner')])
```

Comment:
let's shift that into `McpBundle`, make it conditional, drop the `ai.` prefix in favor of just `mcp.` and use the decoration feature of the container:
```php
if ($container->getParameter('kernel.debug')) {
    $traceableRegistry = (new Definition('mcp.traceable_registry'))
        ->setClass(TraceableRegistry::class)
        ->setArguments([new Reference('.inner')])
        ->setDecoratedService('mcp.registry');
    $container->setDefinition('mcp.traceable_registry', $traceableRegistry);
}
```